### DISCUSSION TITLE
HmIP-ASIR-O Unterstützung

### DISCUSSION THREAD

**Discussion by mplogas** [SukramJ/aiohomematic#734] [Q&A] [answered]
Hi, ich habe eine HmIP Sirene (HMIP-ASIR-O), die bislang via CCU/virtuellem Trigger angesprochen wird. 

Nach der Migration zur custom_homematic Erweiterung habe ich für meine Sirene ein Toggle der aber noch nichts macht

![image](https://user-images.githubusercontent.com/842121/206154657-5842454a-fe38-4ed8-a964-3f9b766bdcc2.png)
 
Bevorzugt würde ich die "Notlösung" via virtuellem Trigger ablösen, was muss für eine vollständige Integration der Sirene in HA noch beachtet werden?

**Comment by SukramJ** [comment, marked as answer]
Doku lesen:
[HA default Service](https://www.home-assistant.io/integrations/siren/#service-sirenturn_on)
[Custom HmIP service](https://github.com/danielperna84/custom_homematic#homematicip_localturn_on_siren)

Die entsprechenden optischen bzw. akustischen Signale findest du in den referenzierten `helper entities` des custom service.

**Comment by mplogas** [comment]
Hi @SukramJ , danke für die Links.

1. Was macht der Toggle denn? Ich kenne die HA Doku zur Sirene und hatte daher erwartet dass der Toggle die Sirene startet
2. Der zweite Link sieht vielversprechend aus. Ich teste das morgen mal im Hellen und melde mich zurück.

**Reply by SukramJ** [reply]
> Ich kenne die HA Doku zur Sirene und hatte daher erwartet dass der Toggle die Sirene startet

Aus der Doku geht sowas nicht hervor. Was passiert denn, wenn die Sirene ausgelöst wurde? Kann man die Sirene damit ausschalten?

**Reply by mplogas** [reply]
So, ich habe den heutigen Warntag genutzt um die Integration der Sirene zu testen. 

Der custom HmIP Service ```homematicip_local.turn_on_siren``` schaltet die Sirene zuverlässig an, der HA Service ```siren.turn_off``` schaltet die Sirene wieder aus. 

> Was passiert denn, wenn die Sirene ausgelöst wurde? Kann man die Sirene damit ausschalten?

Genau! Wenn die Sirene aktiviert ist kann man sie mit dem Toggle wieder ausschalten. Hab ich sehr zur Freude der Nachbarn erfolgreich getestet. War jetzt nicht ganz intuitiv.

Danke nochmals!

**Reply by slackstarr** [reply]
@mplogas Tach!

Da ich noch ein frischer Nutzer von Home Assistant bin, wollte ich dich fragen, wie man die Alarmsirene richtig einbindet. Wenn ich eine neue Kachel erstelle, bleibt der Toggle leider ohne Funktion. Die Dokus habe ich alle gelesen, aber anscheinend fehlt mir noch Bssiswissen.

Besten Dank!

**Reply by mplogas** [reply]
> Der custom HmIP Service homematicip_local.turn_on_siren schaltet die Sirene zuverlässig an, der HA Service siren.turn_off schaltet die Sirene wieder aus.

Entweder du rufst die Services via Buttons oder eben in einer Automatisierung bzw einem Script auf. Schau dir doch mal die Home Assistant Doku dazu an.

**Reply by Isengo1989** [reply]
Hier ein einfaches Script dazu:

```
alias: Alarm
sequence:
  - service: homematicip_local.turn_on_siren
    entity_id: siren.hmip_asir_b1_000ad993948185
    data:
      tone: FREQUENCY_RISING
      light: BLINKING_ALTERNATELY_REPEATING
      duration: 2
mode: single
icon: mdi:alarm-light
```
